app/js/lib/foxbox/recipes.js

```diff
@@ -24,232 +49,222 @@ export default class Recipe {
    * @return {Promise}
    */
   getAll() {
-    return this[p.net].fetchJSON(
-      `${this[p.net].origin}/api/v${this[p.settings].apiVersion}/services`)
-      // Get all recipes.
-      .then(services => services.filter(service =>
-        service.adapter !== 'thinkerbell-adapter'
-          && service.adapter !== 'webpush@link.mozilla.org'
-          && service.adapter !== 'console@link.mozilla.org'
-      ))
-      // Fetch their respective enabled status.
-      .then(recipes => {
-        const promises = recipes.map(recipe => {
-          const payload = [
-            { id: `${recipe.id}/get_enabled` }
-          ];
-
-          return this[p.net].fetchJSON(
-            `${this[p.net].origin}/api/v${this[p.settings].apiVersion}/` +
-            'channels/get',
-            'PUT',
-            payload
-          );
+    let recipeServicesPromise = this[p.net].fetchJSON(
```

Comment:
Ah it's just leftover - I was refactoring step-by-step - will merge them (but eventually we may need to split them in separate methods to read it easier)